README.md

```diff
@@ -44,3 +44,18 @@ To help identify and subsequently fix bugs faster, make sure to include some bas
 ## Contributing Code
 
 Simply create a branch based on the `dev` branch, do your changes, and open it as a pull request. Ideally write tests for the feature or bugfix you propose.
+
+
+## Installing on ubuntu 17 and running tests
+
+```bash
+
+apt-get update
```

Comment:
I think line 59 should be moved to travis.yml instead so travis automatically runs it

Reply:
sure adding php unit to travis is possible.

Reply:
checking this https://docs.travis-ci.com/user/languages/php/#Default-Build-Script it says that php unit will be invoked by default.

Reply:
It seems you overwrote the script so I am adding this in.